yapf/__init__.py

```diff
@@ -204,51 +204,21 @@ def FormatFiles(filenames,
   Returns:
     True if the source code changed in any of the files being formatted.
   """
-  try:
-    import concurrent.futures
-    formatter = _FormatFilesMultiProcess
-  except ImportError:
-    formatter = _FormatFilesSingleProcess
-  return formatter(filenames, lines, style_config, no_local_style, in_place,
-                   print_diff, verify)
-
-
-def _FormatFilesMultiProcess(filenames,
-                             lines,
-                             style_config=None,
-                             no_local_style=False,
-                             in_place=False,
-                             print_diff=False,
-                             verify=True):
-  import concurrent.futures
+  from yapflib.py3compat import ProcessPoolExecutor, as_completed
```

Comment:
No need to do this. Just use `py3compat.ProcessPoolExecutor` and `py3compat.as_completed`.